docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -141,48 +131,48 @@ zk-2      0/1       Running   0         19s
 zk-2      1/1       Running   0         40s
 ```
 
-The StatefulSet controller creates three Pods, and each Pod has a container with 
+The StatefulSet controller creates three Pods, and each Pod has a container with
 a [ZooKeeper](http://www-us.apache.org/dist/zookeeper/stable/) server.
 
 ### Facilitating Leader Election
 
-As there is no terminating algorithm for electing a leader in an anonymous 
-network, Zab requires explicit membership configuration in order to perform 
-leader election. Each server in the ensemble needs to have a unique 
+As there is no terminating algorithm for electing a leader in an anonymous
+network, Zab requires explicit membership configuration to perform
+leader election. Each server in the ensemble needs to have a unique
 identifier, all servers need to know the global set of identifiers, and each
 identifier needs to be associated with a network address.
 
-Use [`kubectl exec`](/docs/user-guide/kubectl/{{page.version}}/#exec) to get the hostnames 
+Use [`kubectl exec`](/docs/user-guide/kubectl/{{page.version}}/#exec) to get the hostnames
 of the Pods in the `zk` StatefulSet.
 
 ```shell
 for i in 0 1 2; do kubectl exec zk-$i -- hostname; done
 ```
 
-The StatefulSet controller provides each Pod with a unique hostname based on its 
-ordinal index. The hostnames take the form `<statefulset name>-<ordinal index>`. 
-As the `replicas` field of the `zk` StatefulSet is set to `3`, the Set's 
-controller creates three Pods with their hostnames set to `zk-0`, `zk-1`, and 
-`zk-2`. 
+The StatefulSet controller provides each Pod with a unique hostname based on its
+ordinal index. The hostnames take the form of `<statefulset name>-<ordinal index>`.
+As the `replicas` field of the `zk` StatefulSet is set to `3`, the Set's
+controller creates three Pods with their hostnames set to `zk-0`, `zk-1`, and
+`zk-2`.
 
 ```shell
 zk-0
 zk-1
 zk-2
 ```
 
-The servers in a ZooKeeper ensemble use natural numbers as unique identifiers, and 
-each server's identifier is stored in a file called `myid` in the server's 
-data directory. 
+The servers in a ZooKeeper ensemble use natural numbers as unique identifiers, and
+stores each server's identifier in a file called `myid` in the server's
```

Comment:
Here you really need to revert to passive voice. "Servers store ..." (not "stores") and at that point you're in a muddle with number. "Servers store ... each server's identifier" doesn't make much sense grammatically or technically. Breaking into multiple sentences might help, too.